modules/openapi-generator/src/test/java/org/openapitools/codegen/AbstractOptionsTest.java

```diff
@@ -54,7 +54,7 @@ public void checkOptionsProcessing() {
 
     @Test(description = "check if all options described in documentation are presented in test case")
     public void checkOptionsHelp() {
-        final List<String> cliOptions = getCodegenConfig().cliOptions().stream().map(getCliOptionTransformer()).collect(Collectors.toList());
+        final List<String> cliOptions = getCodegenConfig().cliOptions().stream().map(CliOption::getOpt).collect(Collectors.toList());
```

Comment:
This was causing a compiler error because it was previously using a `Function` from Guava. Updated to use a native Java 8 `Function`.